### ISSUE TITLE
two errors in timeDomainHRV

### ISSUE THREAD

**Issue by SCUnzq** [jramshur/HRVAS#10]
![20190313125950](https://user-images.githubusercontent.com/33393139/54254777-0a2f7400-4590-11e9-96f3-8d64b5cf7493.jpg)
1.SDNNi,SDANN functions seem not  to be suitable for their variables .
2. The second nBins seems not  to be commented.
Am I right ?

**Comment by jramshur** [comment]
Good catch. You may be correct about the SD variables. I honestly can't remember why I changed nBins to 32. I must have decided to fix it at 32 bins at some point during experimenting and just never formally documented correctly in code.
